<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTollDiscountTypeSet</class>
 <widget class="QFrame" name="CTollDiscountTypeSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收费折扣种类设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>641</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QTableWidget" name="tableTollDiscount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <column>
     <property name="text">
      <string>折扣名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>计算方法</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>折扣数值</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>说明</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>index</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>621</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnModify">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMinimalize">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>601</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>说明：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtExplain">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>折扣名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtName">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>折扣计算方法：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxMethod">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <item>
         <property name="text">
          <string>按固定金额抵扣</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按百分比计算</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
        <property name="text">
         <string>折扣数值：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spValue">
        <property name="styleSheet">
         <string notr="true">background-image:url(none);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lblID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
